.github/resources/.actions/cli-runner/action.yml

```
@@ -0,0 +1,130 @@
+name: Slack CLI Installation and Command Runner 
+description: Download and cache the Slack CLI and run the input command 
+
+inputs:
+  command:
+    description: "Slack CLI command to run"
+    type: string
+    default: ""
+    required: true
+  verbose:
+    description: "Verbose flag"
+    type: boolean
+    default: false
+    required: false
+  cli_version:
+    description: "CLI Version"
+    type: string
+    default: "latest"
```

```suggestion
    default: "3.x.x"
```

🚀 suggestion(non-blocking): I'm hoping we can soon land floating version in https://github.com/slackapi/slack-cli/pull/199! For now the installation script should default to the **latest** without such specifier though.